environment

```diff
@@ -63,7 +63,11 @@ DSS_GS_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
 DSS_NOTIFICATION_SENDER="DSS Notifier <dss.humancellatlas@gmail.com>"
 ADMIN_USER_EMAILS=""
 API_DOMAIN_NAME=dss.dev.data.humancellatlas.org
-TOKENINFO_URL=https://${API_DOMAIN_NAME}/internal/tokeninfo
+
+# TOKEN_INFO_URLs naming is intentionally different than TOKENINFO_FUNC to prevent connexion from using it.
+TOKEN_INFO_URL=https://humancellatlas.auth0.com/userinfo
```

Comment:
As per the above, I don't think we should be using the tokeninfo API at all. Please remove references to it.